### PR TITLE
feat: go and travis support, ignores interface- badges

### PR DESCRIPTION
**Pull request by hacdias**

### CODE REVIEW

File: src/badges.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const name = (gh, pkg) => {
+  return `[\`${pkg}\`](//github.com/${gh})`
+}
+
+const version = (gh, pkg) => {
```
Comment (daviddias) [inline src/badges.js]: perhaps call tis npmVersion

**Review by daviddias — APPROVED** [review]

**Comment by hacdias** [comment]
@diasdavid added the badge existence verification for CIs. It's very async-y. ahah

**Review by daviddias — APPROVED** [review]

**Comment by hacdias** [comment]
@diasdavid you can now publish a new version!

**Comment by daviddias** [comment]
@hacdias you have publish access too, go for it :)

**Comment by hacdias** [comment]
0.3.0 📦

**Pull request merged**